### ISSUE TITLE
Furaffinity links not working.

### ISSUE THREAD

**Issue by IkeFanboy64** [JeremyRuhland/saucebot-discord#29]
Dunno if there's an issue with permissions, but the bot does not seem to be working for FA links on both mine and my friend's servers.

**Comment by RestrainedRaptor** [comment]
Yeah, I just came here to check that myself. It stopped working for DA as well.

This is probably due to Discord's changes to limit bots that haven't updated to use slash commands. If you @ the bot directly, it will work. From a recent announcement:

> If your server uses a verified app, the way you send commands has probably changed in the last year - if it hasn’t, it may very soon! Developers of verified apps have been migrating to Slash Commands. The deadline to finish migrating is the end of August, which means on Sept 1, any apps that haven’t finished may have reduced functionality.
